### ISSUE TITLE
Review of todo app.

### ISSUE THREAD

**Issue by mehtanirav** [mehtanirav/ToDo-Android-App#1]
/cc @codepathreview @codepath

**Comment by mehtanirav** [comment]
I will work on extending the todo app with more functionalities.

**Comment by nesquena** [comment]
Hi Nirav,

Thanks for your interest. This public course at Uber had 5 internal engineers selected by Akash Garg to participate. I believe those 5 have already been selected. Outside of those selected by Akash, the remaining seats are reserved for engineers not currently employed at Uber. Let me know if you were already notified that you were one of the 5 selected by Akash. Otherwise, hope you are able to participate in a future cohort. Sorry about any confusion.

**Comment by mehtanirav** [comment]
Hi Nathan,

I knew that only 5 internal engineers at Uber will be joining, but somehow didn't know that they were already selected. I will lookout for future batches to join.

Cheers!
Nirav

**Comment by nesquena** [comment]
Yep, no worries. I'll let Akash know about your interest in participating in a future course.

**Comment by codepathreview** [comment]
Hello Nirav,

Looks good, this exercise is intended in part to give you an introduction to the general rhythm of this course. The course is entirely project-based with an app being assigned each week and then due the following week. Each project builds on the last to help each engineer  learn the practical elements of Android development and best practices as quickly as possible. We also do a code review for each submitted project once the program begins.

The next step is to continue working on [extensions to your todo app](http://courses.codepath.com/snippets/intro_to_android/prework#heading-5-extending-your-todo). With regards to extensions to implement, we are looking for developers with an interest in user experience and product development so be sure to focus on additional functional enhancements and/or improving the user interface of the app.

We'll be following up with you again shortly to outline the next steps in the admissions process.

**Comment by nesquena** [comment]
@mehtanirav Hi Nirav, wanted to check in! Are you still planning on extending your pre-work?

**Comment by mehtanirav** [comment]
Hi Nathan,

I haven't gotten around at working on the extensions to the pre-work. I plan to do some work next week. Is there a deadline for the extensions to pre-work?

Thanks!
Nirav

**Comment by nesquena** [comment]
Okay thanks for the heads up. Yes, deadline is the 30th.

**Comment by mehtanirav** [comment]
Thanks. I will make sure to add some extensions before that.
